js/holmes.js

```diff
@@ -279,9 +282,15 @@
 
       // loop over all the elements
       // in case this should become dynamic, query for the elements here
-      holmes.prototype.regex = new RegExp('(' + holmes.prototype.searchString + ')(?![^<]*>)', 'gi');
+      if (holmes.prototype.options.mark) {
+        holmes.prototype.regex = new RegExp('(' + holmes.prototype.searchString + ')(?![^<]*>)', 'gi');
+      }
 
-      holmes.prototype.elementsArray.forEach(function (element) {
+      holmes.prototype.elementsArray.forEach((function (element) {
+        // still typing
```

Comment:
It works when user typing.
When prevous searchString like current searchString.

For example:
`prev` -> `prevo` -> `prevou` -> `prevous`